server/db.go

```diff
@@ -529,7 +531,7 @@ func (c *Server) fetchIssuers(issuerType string) ([]Issuer, error) {
 // FetchAllIssuers fetches all issuers from a cache or a database, saving them in the cache
 // if it has to query the database.
 func (c *Server) FetchAllIssuers() (*[]Issuer, error) {
-	if cached := retrieveFromCache(c.caches, "issuers", "all"); cached != nil {
+	if cached := retrieveFromCache(c.caches, "issuers", "all", 0); cached != nil {
```

Comment:
This reads as if you're only fetching all issuers in cohort 0. However I can see from the implementation below that the behaviour of `retrieveFromCache` is that it ignores the cohort if a value with the undecorated key is found. 

It would be considerably clearer if the cohort parameter was not passed with a valid value in this case.

Reply:
In that case, I'd need a configuration variable to trigger the behavior, as Grants does not have this suffix issue and shares this code.